app/screens/channel/header/channel_banner/channel_banner.tsx

```diff
@@ -48,6 +48,9 @@ const getStyleSheet = (bannerTextColor: string) => ({
         flex: 1,
         flexGrow: 1,
     },
+    bannerText: {
+        textAlign: 'center' as const,
```

Comment:
do we need 'as const'?

Reply:
yes, otherwise TS complaints, source - https://stackoverflow.com/questions/43121661/typescript-type-inference-issue-with-string-literal